### PR TITLE
Fix some typos and omissions in kubeadm APIs

### LINKED ISSUE #102826: typo in kubeadm api doc: "node subnet" instead of "pod subnet"
#### What happened:
I was reading about how to use kubeadm and noticed a typo in the API documentation.  https://github.com/kubernetes/kubernetes/blob/52eea971c57580c6b1b74f0a12bf9cc6083a4d6b/cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go#L123 speaks of the "node subnet" where I think "pod subnet" is intended.

#### Environment:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### LINKED ISSUE #102827: syntax of KubeletExtraArgs is not clear
#### What happened:
I was reading the doc about NodeRegistration, with an interest in specifying some `kubeletExtraArgs`.  I noticed that the doc (e.g., https://github.com/kubernetes/kubernetes/blob/52eea971c57580c6b1b74f0a12bf9cc6083a4d6b/cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go#L207-L210) does not tell me exactly how to identify each arg: exactly as on the command line (including leading dash(es)), as on the command line but without leading dash(es), or camelCase.

#### What you expected to happen:
I expected the doc to enable me to write some `kubeletExtraArgs` with confidence.

#### Environment:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
This fixes a typo in the kubeadm API docs and adds some helpful information about how to identify xxxExtraArgs.

#### Which issue(s) this PR fixes:

Fixes #102826 
Fixes #102827 

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
@kubernetes/sig-cluster-lifecycle

File: cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ limitations under the License.
 // including settings for:
 //
 // - Networking, that holds configuration for the networking topology of the cluster; use it e.g. to customize
-// node subnet or services subnet.
+// pod subnet or services subnet.
```
Comment (SataQiu) [inline cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go:124]: Good catch! @MikeSpreitzer 
/approve

File: cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go [hunk 2]
```diff
@@ -116,6 +116,8 @@ type ClusterConfiguration struct {
 // ControlPlaneComponent holds settings common to control plane component of the cluster
 type ControlPlaneComponent struct {
 	// ExtraArgs is an extra set of flags to pass to the control plane component.
+	// The keys in this map are the flag name as it appears on the
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go:119]: Plural vs singular:
Should this be 'A key in this map is a flag...'

Comment (MikeSpreitzer) [reply cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go]: Thanks, fixed.

**Comment by neolit123** [comment]
could you please update the "extra arg" comments in the internal type to not have a drift, even if these are not user facing:
https://github.com/kubernetes/kubernetes/blob/3946a9154c1e913834509f5cc9eb45defb218721/cmd/kubeadm/app/apis/kubeadm/types.go

also please keep the commits squashed to one.

**Review by neolit123 — COMMENTED** [review]
Thanks
/lgtm

**Pull request merged**